tests/components/air_quality/__init__.py

```diff
@@ -0,0 +1 @@
+"""The tests for Air Quality platforms."""
```

Comment:
Was the renamed modules moved with `git mv` one module per commit?

Reply:
No, moved them manually :(

Reply:
Try to use `git mv` one module at a time, and commit between each time that is done. Hopefully that will keep history intact. Sometimes github doesn't register git mv correctly though.